flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -129,6 +129,13 @@ class JobManager(
   var leaderSessionID: Option[UUID] = None
 
   /**
+   * The port of the web monitor as configured. Make sure that it is actually configured before
+   * starting the JobManager.
+   */
+  val webMonitorPort : Integer = flinkConfiguration.getInteger(
```

Comment:
Maybe we can add a comment which says that this is gonna be kicked out once we have made the `ExecutionGraph` properly serializable or so...